test-suite/output/PrintAssumptions.out

```diff
@@ -10,16 +10,16 @@ Axioms:
 seq relies on definitional UIP.
 Axioms:
 extensionality
-  : forall (P Q : Type) (f g : P -> Q), (forall x : P, f x = g x) -> f = g
+   : forall (P Q : Type) (f g : P -> Q), (forall x : P, f x = g x) -> f = g
```

Comment:
Is this change intended? No big deal, but I feel like it was fine before.

Reply:
It is unrelated to the other changes but it was intended in the commit. This was the only case indenting with 1 space while other indentations are using 2 spaces.

But I agree that 1 is esthetically fine also, so I can revert if you prefer.

Reply:
Actually, you're right, I made a mistake. There is already one space before ":", so no need for one more. Will change when full CI is finished.